밍붕-김민서/3주차 미션 밍붕/src/pages/movies.jsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import { useParams } from 'react-router-dom';
+
+const MoviesPage = () => {
+    const [movies, setMovies] = useState([]);
+    let { id } = useParams();
+    id = id || 'popular';
```

Comment:
여기서 id가 falsy한 값을 띄면 'popular'가 할당이 될 거여서 const으로 하는 것이 좋습니다! 혹시 이름이 겹쳐서 고민이시라면 아래처럼도 가능해요!
```suggestion
  const { id: paramId } = useParams();
  const id = paramId || "popular";
```

Reply:
감사합니다

Reply:
+ let은 지양하시는게 좋지만 사용하게 된다면 특정 함수 내부에서 쓰는 등 specific한 scope에서만 사용하시는 것을 권장드립니다! let에 새로운 값이 할당되는 것을 리액트가 알 수가 없어서 일관된 라이프 사이클 밖에서 움직일 수 있게 되거든요!